filebeat/config/config.go

```diff
@@ -98,13 +99,13 @@ func mergeConfigFiles(configFiles []string, config *Config) error {
 }
 
 // Fetches and merges all config files given by configDir. All are put into one config object
-func (config *Config) FetchConfigs() {
+func (config *Config) FetchConfigs() error {
```

Comment:
Thanks for the cleanup here. This part could probably go into a separate PR to keep the changes in this one to a minimum.